commands/user_e2e_test.go

```diff
@@ -136,3 +136,36 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestUpdateUserEmailCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForSystemAdminAndLocal("admin and local can change basic user email", func(c client.Client) {
+		newEmail := "basicuser2@fakedomain.com"
+		err := updateUserEmailCmdF(c, &cobra.Command{}, []string{s.th.BasicUser2.Email, newEmail})
+		s.th.BasicUser2.Email = newEmail
+		s.Require().Nil(err)
+	})
+
+	s.Run("normal client doesn't have permission to change email of another client", func() {
+		s.th.LoginBasic2WithClient(s.th.Client)
+		newEmail := "basicuser-change@fakedomain.com"
+		err := updateUserEmailCmdF(s.th.Client, &cobra.Command{}, []string{s.th.BasicUser.Id, newEmail})
+		s.Require().EqualError(err, ": You do not have the appropriate permissions., ")
+		s.th.BasicUser.Email = newEmail
+	})
+
+	s.RunForAllClients("wrong number of arguments", func(c client.Client) {
+		printer.Clean()
+
+		err := updateUserEmailCmdF(c, &cobra.Command{}, []string{})
+		s.Require().EqualError(err, "expected two arguments. See help text for details")
+	})
+
+	s.RunForAllClients("not valid email", func(c client.Client) {
+		printer.Clean()
+
+		err := updateUserEmailCmdF(c, &cobra.Command{}, []string{s.th.BasicUser.Email, "not-a-valid-email"})
+		s.Require().EqualError(err, "invalid email: 'not-a-valid-email'")
+	})
+}
```

Comment:
I'm only missing the test case in which an unprivileged user changes its own password. Would you be open to add that one?